server/sonar-server/src/test/java/org/sonar/server/qualityprofile/RuleActivatorMediumTest.java

```diff
@@ -1138,4 +1148,10 @@ private void verifyZeroActiveRules(String key) {
         .setQProfileKey(key)
         .setActivation(true))).isEmpty();
   }
+
+  private void assertProfileHasBeenUpdatedManually(String profileKey) {
```

Comment:
You should add a test where you check that profile is not manually updated when it was not updated by a user